README.md

```diff
@@ -4,7 +4,7 @@
 [![Version](https://img.shields.io/jetbrains/plugin/v/15347-djinni-idl-file-support.svg)](https://plugins.jetbrains.com/plugin/15347-djinni-idl-file-support)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/15347-djinni-idl-file-support.svg)](https://plugins.jetbrains.com/plugin/15347-djinni-idl-file-support)
 
-:arrow_right: Documentation: [djinni.xlcpp.dev](https://djinni.xlcpp.dev/djinni-intellij-plugin/about/)
+:arrow_right: Documentation: [djinni.xlcpp.dev](https://djinni.xlcpp.dev/djinni-intellij-plugin/)
```

Comment:
The Url has changed because I renamed `about.md` to `index.md`. I like this a lot more.